3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.iOS/Info.plist

```diff
@@ -24,9 +24,11 @@
 	<key>IntuneMAMSettings</key>
 	<dict>
 		<key>ADALClientId</key>
```

Comment:
@SameerK-MSFT - Given recent discussion with the MSAL team, I think we should add ADALAuthority setting here to make sure single-tenant apps work.

Reply:
Added